week06_policy_based/a2c-optional.ipynb

```diff
@@ -13,8 +13,10 @@
      "    # Undo the recent upgrade of importlib-metadata\n",
      "    ## Thanks to @Mxhiu: https://stackoverflow.com/a/73932581/14473118\n",
      "    os.system(\"pip install -q importlib-metadata==4.13.0\")\n",
-     "    os.system('apt-get install -y xvfb')\n",
-     "    os.system('wget https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/xvfb -O ../xvfb')\n",
+     "    # Download and install Video game dependencies\n",
+     "    os.system('wget -q https://raw.githubusercontent.com/yandexdataschool/Practical_RL/master/setup_colab.sh -O- | bash')\n",
+     "    os.system('touch .setup_complete')\n",
```

Comment:
It is a bit weird the file is placed because this is what I have in my repos!!

![image](https://user-images.githubusercontent.com/72295771/205710308-82ef8c6d-c9eb-4885-864e-e8d19bca333e.png)

Maybe this is an old version...